config/main.py

```diff
@@ -3619,8 +3619,12 @@ def neighbor(ipaddr_or_hostname, verbose):
         if _change_bgp_session_status(config_db, ipaddr_or_hostname, 'down', verbose):
             found_neighbor = True
 
+    device_metadata = config_db.get_entry('DEVICE_METADATA', 'localhost')
     if not found_neighbor:
-        click.get_current_context().fail("Could not locate neighbor '{}'".format(ipaddr_or_hostname))
+        if device_metadata['type'] == 'SpineRouter':
```

Comment:
Is there a way to check for "Chassis" instead of just "SpineRouter"?  Reason is for T1 it is also "SpineRouter" and for this case it would not be good to say it is expected right?

Reply:
never mind.. got clarified with @abdosi T1 is "LeafRouter" so we are ok with this check.